w3w-autosuggest/public/class-w3w-autosuggest-public.php

```diff
@@ -214,51 +214,51 @@ public function add_fields_to_checkout_form( $fields ) {
         'clear' => true
       ];
 
-      if ( $settings['save_nearest_place'] ) {
-
-        $fields['billing']['billing_nearest_place'] = [
-          'type' => 'hidden',
-          'required' => false,
-          'clear' => true,
-          'class' => ['hidden', 'form-row-wide'],
-        ];
-        $fields['shipping']['shipping_nearest_place'] = [
-          'type' => 'hidden',
-          'required' => false,
-          'clear' => true,
-          'class' => ['hidden', 'form-row-wide'],
-        ];
-
-      }
-
-      if ( $settings['return_coordinates'] ) {
-
-        $fields['billing']['billing_w3w_lat'] = [
-          'type' => 'hidden',
-          'required' => false,
-          'clear' => false,
-          'class' => ['hidden', 'form-row-wide']
-        ];
-        $fields['billing']['billing_w3w_lng'] = [
-          'type' => 'hidden',
-          'required' => false,
-          'clear' => false,
-          'class' => ['hidden', 'form-row-wide']
-        ];
-        $fields['shipping']['shipping_w3w_lat'] = [
-          'type' => 'hidden',
-          'required' => false,
-          'clear' => false,
-          'class' => ['hidden', 'form-row-wide']
-        ];
-        $fields['shipping']['shipping_w3w_lng'] = [
-          'type' => 'hidden',
-          'required' => false,
-          'clear' => false,
-          'class' => ['hidden', 'form-row-wide']
-        ];
-
-      }
+    }
```

Comment:
Important part is the previous if statement ending earlier than before - in the mode where the autosuggest is applied to an existing field, the `woocommerce_enabled` flag is `false` despite still using the woocommerce platform. The extra hidden fields are added in all cases now, which hopefully won't be an issue for other platforms.